daemon/logger/journald/read.go

```diff
@@ -220,10 +220,6 @@ func (s *journald) readLogs(logWatcher *logger.LogWatcher, config logger.ReadCon
 	}
 	// Add a match to have the library do the searching for us.
 	cmatch = C.CString("CONTAINER_ID_FULL=" + s.vars["CONTAINER_ID_FULL"])
```

Comment:
I might be looking the wrong thing, but on  https://golang.org/cmd/cgo/ I see:
```
func C.CString(string) *C.char
```
so shouldn't we check for nil  since its a pointer?